packages/language-server/test/snapshots/xml-view-diagnostics/utils.ts

```diff
@@ -0,0 +1,106 @@
+import { resolve, sep } from "path";
```

Comment:
Shouldn't this file be named `snapshot-utils` if it contains utilities for the snapshot tests? It's more specific and will prevent adding unrelated functions to it

Reply:
👍